content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+## Proposal {.page_break_before}
+
+Manubot is a system for writing scholarly manuscripts on GitHub.
+Manubot aims to **transform publishing** to be transparent & reproducible, immediate & permissionless, versioned & automated, collaborative & open, linked & provenanced, decentralized & hackable, interactive & annotated, and free of charge.
+Wooh!
+
+Manubot applies **tactics from open source** software to manuscript writing.
+Manuscripts are written in markdown, using git for version control and GitHub for collaboration and review.
+Continuous integration automatically rebuilds and deploys the manuscript when it changes [@doi:10.1038/nbt.3780; @doi:10.1038/550143a].
+Manubot provides many state-of-the-art features.
+For example, users cite standard identifiers, while bibliographic details are automatically retrieved.
```

Comment:
With the removal above: Users can cite standard persistent identifiers and manubot retrieves bibliographic details to automatically create a reference list.

Reply:
Done in https://github.com/greenelab/manufund-2018/pull/2/commits/65d24ce1435ad83ef0ab995aa5e7432766465208